user.go

```diff
@@ -1,3 +1,57 @@
-package spotify
-
-type UserService service
+package spotify
+
+import (
+	"fmt"
+)
+
+type UserService service
+
+type SpfError struct {
+	Error `json:"error"`
+}
+
+type Error struct {
+	Status  int    `json:"status"`
+	Message string `json:"message"`
+}
+
+type ResultProfile struct {
+	DisplayName  string      `json:"display_name"`
+	ExternalUrls spotifyUri  `json:"external_urls"`
+	Followers    dataFlowers `json:"followers"`

```

Comment:
Use Followers type instead of dataFlowers

Reply:
👍